### PR TITLE
fix csv export data bugs

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
Relates #445

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/pages/EarnGroup/Members/AllMembersLogic.js [hunk 1]
```diff
@@ -36,16 +36,18 @@ const EarnGroupMembers = (props) => {
 
   const CSVEarnGroupMembersHeaders = [
     { label: "Member ID", key: "memberId" },
```
Comment (thejoefriel) [inline client/src/pages/EarnGroup/Members/AllMembersLogic.js:38]: Remove this label and data point. We don't want them to have the memberID

File: client/src/pages/EarnGroup/Members/AllMembersLogic.js [hunk 2]
```diff
@@ -36,16 +36,18 @@ const EarnGroupMembers = (props) => {
...
-    { label: "Name", key: "member" },
+    { label: "First Name", key: "firstName" },
     { label: "Last Name", key: "lastName" },
     { label: "Join Date", key: "date" },
```
Comment (thejoefriel) [inline client/src/pages/EarnGroup/Members/AllMembersLogic.js:41]: Need this to only be the date, not the time

File: client/src/pages/EarnGroup/Members/AllMembersLogic.js [hunk 3]
```diff
@@ -36,16 +36,18 @@ const EarnGroupMembers = (props) => {
...
     { label: "Time Credits earned", key: "creditsEarned" },
     { label: "% Time Credit earned that have been spent", key: "creditsSpent" },
     { label: "Member status", key: "status" },
+    { label: "DOB (non digital)", key: "birthDate" },
```
Comment (thejoefriel) [inline client/src/pages/EarnGroup/Members/AllMembersLogic.js:45]: Sorry please remove this. They were only referring to Date of Birth being in the export of members that Tempo Staff do

File: client/src/pages/EarnGroup/Members/AllMembersLogic.js (hunk 1 above)
Comment (RamyAlshurafa) [reply client/src/pages/EarnGroup/Members/AllMembersLogic.js:38]: done

File: client/src/pages/EarnGroup/Members/AllMembersLogic.js (hunk 3 above)
Comment (RamyAlshurafa) [reply client/src/pages/EarnGroup/Members/AllMembersLogic.js:45]: done

File: client/src/pages/EarnGroup/Members/AllMembersLogic.js (hunk 2 above)
Comment (RamyAlshurafa) [reply client/src/pages/EarnGroup/Members/AllMembersLogic.js:41]: done

**Pull request merged**